src/group/mls_group/create_commit.rs

```diff
@@ -97,16 +95,11 @@ impl MlsGroup {
         // Calculate tree hash
         let tree_hash = provisional_tree.tree_hash();
 
-        // TODO #186: Implement extensions
-        let extensions: Vec<Box<dyn Extension>> = Vec::new();
-
         // Calculate group context
-        let provisional_group_context = GroupContext::new(
-            self.group_context.group_id.clone(),
-            provisional_epoch,
-            tree_hash.clone(),
+        let provisional_group_context = GroupContext::from_previous_group_context(
```

Comment:
Looking at how this is used here. Would it make sense to have a `ProvusionalGroupContext` struct instead that only stores the necessary changes and/or references?